### ISSUE TITLE
Link without href does not get highlighted in high contrast

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#789]
![image](https://cloud.githubusercontent.com/assets/11937034/21847360/83793974-d821-11e6-9ff6-15a6b85d3f44.png)

The screen shot is taken from the fabric-react demo page.

**Comment by cschlechty** [comment]
By design. If it's not focusable, the browser treats it as text.

**Comment by cschlechty** [comment]
Since link with no href is a button, additional css is needed to make it "link-like."

**Comment by leddie24** [comment]
Closing issue.  We're rendering Links as buttons if they don't have a href attribute, cannot fix by design.
